utils/python/CIME/XML/generic_xml.py

```diff
@@ -14,17 +14,8 @@ def __init__(self, infile=None):
         """
         Initialize an object
         """
-        self.tree = None
-
-        # Hold arbitary values. In create_newcase we may set values
-        # for xml files that haven't been created yet. We need a place
-        # to store them until we are ready to create the file. At file
-        # creation we get the values for those fields from this lookup
-        # table and then remove the entry. This was what I came up
-        # with in the perl anyway and I think that we still need it
-        # here.
-        self.lookups = {}
-        self.lookups['CIMEROOT'] = get_cime_root()
+        if not hasattr(self, 'tree'):
```

Comment:
Look at __init__ in machines.py to see how this is used.   I could avoid calling the generic __init__ the second time and just call read - that probably makes more sense, thanks.